final_prep/annotations.rb

```diff
@@ -4,10 +4,16 @@
 
 # Build a Bear
 
+# create a method with multiple parameters
 def build_a_bear(name, age, fur, clothes, special_power)
+# interpolation of name in greeting string value
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+# variable defined to combine dynamic variables name and age
   demographics = [name, age]
+# dynamic variable that takes argument passed to it through object instance (not 100% sure bc I don't see a class creation here)
```

Comment:
I'm glad you noted you weren't 100% clear - 

The variable `special_power` holds the value that was passed into the method, up on line 8.